django/permissions/README.md

```diff
@@ -0,0 +1,105 @@
+# Permissions
+
+* Keep permissions inside `permissions.py`
+* Check [third-party packages](https://www.django-rest-framework.org/api-guide/permissions/#third-party-packages) before writing your own permission classes
+* Function-based views will need to check object permissions explicitly by calling `.check_object_permissions(request, obj)`, raising `PermissionDenied` on failure.
+
+## Default permission policy
+
+Always provide the default permission classes in `settings.py`
+
+```python
+# settings.py
+
+REST_FRAMEWORK = {
+    'DEFAULT_PERMISSION_CLASSES': [
+        'rest_framework.permissions.IsAuthenticated',
+    ]
+}
+```
+
+## Example permission class
+
+```python
+# permissions.py
+
+from rest_framework import permissions
+
+
+class ReadOnly(permissions.BasePermission):
+    def has_permission(self, request, view):
+        return request.method in permissions.SAFE_METHODS
+
+```
+
+## Compose permissions
+
+Many permission classes can be composed Python bitwise operators.
+*Note: it supports & (and), | (or) and ~ (not).*
```

Comment:
```suggestion
*Note: It supports `&` (and), `|` (or) and `~` (not).*
```

What do you think krubb?

Reply:
Both is correct krub `Note: lowercase` and `Note: Uppercase`